src/sentry/static/sentry/app/views/releasesV2/list/releaseHealth.tsx

```diff
@@ -333,4 +346,11 @@ const ChartWrapper = styled('div')`
   }
 `;
 
+const StyledPlaceholder = styled(Placeholder)`
+  height: 20px;
```

Comment:
Do you need this height if each component also sets the height to 25px?

Reply:
I removed the height props (it was a leftover from the previous iteration)